doc/classes/GraphNode.xml

```diff
@@ -260,6 +260,11 @@
 				Emitted when the GraphNode is moved.
 			</description>
 		</signal>
+		<signal name="slot_updated">
+			<description>
+				Emitted when the any GraphNode's slot is updated.
```

Comment:
This sentence is not grammatically correct 🙂

Reply:
Agree. My mistakes. (^^")